src/scripts/tweaks/hide_note_count_types.js

```diff
@@ -0,0 +1,14 @@
+import { keyToCss } from '../../util/css_map.js';
+import { buildStyle } from '../../util/interface.js';
+
+const styleElement = buildStyle();
+
+export const main = async function () {
+  const selector = await keyToCss('noteCountTypes');
+  styleElement.textContent = `${selector} { display: none !important; }`;
+  document.head.append(styleElement);
+};
+
+export const clean = async function () {
+  styleElement.remove();
+};
```

Comment:
lack of top-level await support in eslint 7 makes april a sad girl, but until eslint-config-standard gets updated we can at least do shit like this

```suggestion
const styleElement = buildStyle();
keyToCss('noteCountTypes').then(selector => { styleElement.textContent = `${selector} { display: none !important; }`; });

export const main = async ()  => document.head.append(styleElement);
export const clean = async () => styleElement.remove();
```

(if this looks worse to you feel free to just resolve this conversation without committing it)

Reply:
I'm for it! Glad to see that rather than co-author for writing the script I copy-pasted, you can get co-author for improving the parts that I copy-pasted :P